pkg/antctl/command_definition.go

```diff
@@ -120,6 +128,8 @@ type commandDefinition struct {
 	// from the handler, it must returns an interface which has same type as
 	// TransformedResponse.
 	AddonTransform func(reader io.Reader, single bool) (interface{}, error)
+	// Flags is a list of all needed flags for this command.
```

Comment:
I think you mean "all possible flags" and not "all needed flags", unless the flags are always mandatory.

Reply:
Yes, thank you!. Updated